Readme.md

```diff
@@ -85,4 +91,6 @@ Xamarin.Forms MacOS
 <img src="https://raw.githubusercontent.com/DreamTeamMobile/Xamarin.Agora.Samples/master/Screenshots/Forms/AgoraXamarinFormsMacOS_00.png" width="500" />
 <img src="https://raw.githubusercontent.com/DreamTeamMobile/Xamarin.Agora.Samples/master/Screenshots/Forms/AgoraXamarinFormsMacOS_01.png" width="500" />
 
+RTM example for Android, iOS and Mac
```

Comment:
Please, make possible to run RTC sample with only text token, without server, and RTM - only with server